palettable/cubehelix/cubehelix.py

```diff
@@ -0,0 +1,409 @@
+            start = (start_hue / 360. - 1.) * 3.
+            rotation = end_hue / 360. - start / 3. - 1.
+
+        # lambd is effectively the color map grid
+        lambd = np.linspace(min_light, max_light, n)
+
+        # apply the gamma correction
+        lambd_gamma = lambd ** gamma
+
+        # Rotation angle
+        # NOTE the equation for phi in Green 2011 does not have an extra `+1`
+        # but the Fortran code does, as does the original cubehelix.py
+        # I'm leaving out the +1 to keep to the original equation, but
+        # worth investigating. In practice I see no difference!
+        phi = 2.0 * np.pi * (start / 3.0 + rotation * lambd)
+
+        if sat is None:
+            sat = np.linspace(min_sat, max_sat, n)
+
+        # Amplitude of helix from grayscale map
+        amp = sat * lambd_gamma * (1. - lambd_gamma) / 2.
+
+        # Compute the RGB vectors according to Green 2011 Eq 2
+        rot_matrix = np.array([[-0.14861, +1.78277],
+                               [-0.29227, -0.90649],
+                               [+1.97294, 0.0]])
+        sin_cos = np.array([np.cos(phi), np.sin(phi)])
+        rgb = (lambd_gamma + amp * np.dot(rot_matrix, sin_cos)).T * 255.
+
+        # Clipping is necessary in some cases when sat > 1
+        np.clip(rgb, 0., 255., out=rgb)
+
+        if reverse:
+            rgb = rgb[::-1, :]
+
+        colors = rgb.astype(int).tolist()
+        return cls(name, colors)
+
+
+def print_maps():
+    """
+    Print a list of pre-made Cubehelix palettes.
+
+    """
+    namelen = max(len(name) for name in palette_names)
+    fmt = '{0:' + str(namelen + 4) + '}{1:16}'
+
+    for name in palette_names:
+        print(fmt.format(name, palette_type))
+
+
+def get_map(name, n=256, reverse=False):
+    """
+    Get a pre-made Cubehelix palette by name.
+
+    Parameters
+    ----------
+    name : str
+        Name of map. Use `print_maps` to see available names. If `None`, then
+        return a list of all colormaps.
+    n : int, optional
+        Number of colours in the palette. Default is ``256``, useful for
+        making continuous color maps.
+    reverse : bool, optional
+        If True reverse colors from their default order.
+
+    Returns
+    -------
+    palette : Cubehelix
+
+    """
+    try:
+        # Make everthing lower case for matching
+        index = [s.lower() for s in palette_names].index(name.lower())
+    except KeyError:
+        msg = "{0!r} is an unknown CubeHelix palette."
+        raise KeyError(msg.format(name))
+
+    real_name = palette_names[index]
+    colors = palette_rgb[real_name]
+
+    if reverse:
+        real_name = real_name + '_r'
+        colors = list(reversed(colors))
+
+    return Cubehelix(real_name, colors)
+
+
+def _get_all_maps():
+    """
+    Returns a dictionary of all Cubehelix palettes, including reversed ones.
+
+    These default palettes are rendered with 256 colours, making them useful
```

Comment:
This is now 16 colors, but it's not really important to bring up since any sequential or diverging palette from palettable is suitable for continuous maps because matplotlib takes care of interpolation.